actors/builtin/reward/testing.go

```diff
@@ -0,0 +1,32 @@
+package reward
+
+import (
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v2/actors/util/math"
+	"github.com/filecoin-project/specs-actors/v2/actors/util/adt"
+)
+
+type StateSummary struct{}
+
+var FIL = big.NewInt(1e18)
+var StorageMiningAllocationCheck = big.Mul(big.NewInt(1_100_000_000), FIL)
+
+func CheckStateInvariants(st *State, store adt.Store, priorEpoch abi.ChainEpoch, balance abi.TokenAmount) (*StateSummary, *builtin.MessageAccumulator, error) {
+	acc := &builtin.MessageAccumulator{}
+
+	acc.Require(big.Add(st.TotalStoragePowerReward, balance).Equals(StorageMiningAllocationCheck), "reward given + reward left != storage mining allocation")
```

Comment:
I don't think we can make an `==` check on the balance; anyone could send funds there by accident. But can check `>=`.

Reply:
Yes we observed this not holding on testnet as it turns out people sent 65 FIL to reward actor.